docset/winserver2022-ps/appx/Add-AppxVolume.md

```diff
@@ -20,28 +21,32 @@ Add-AppxVolume [-Path] <String[]> [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Add-AppxVolume** cmdlet adds an **AppxVolume** for the Package Manager to advertise.
-After you add a volume, appx deployment operations can use that volume as a target.
-This cmdlet returns the volume that it adds.
-Note, the *Path* parameter must be specified as a drive letter followed by "WindowsApps" as the directory.
-Not using the aforementioned format could lead to inconsistent behavior in the application model subsystems or the volume itself; for more information see the examples section.
+
+The **Add-AppxVolume** cmdlet adds an **AppxVolume** for the Package Manager to advertise. After you
+add a volume, appx deployment operations can use that volume as a target. This cmdlet returns the
+volume that it adds. Note, the *Path* parameter must be specified as a drive letter followed by
+"WindowsApps" as the directory. Not using the aforementioned format could lead to inconsistent
+behavior in the application model subsystems or the volume itself; for more information see the
+examples section.
```

Comment:
We always bold parameter names. If a value in the prose is specific for entry in a commandline, we want to backtick those values too.

```suggestion
The `Add-AppxVolume` cmdlet adds an **AppxVolume** for the Package Manager to advertise. After you
add a volume, appx deployment operations can use that volume as a target. This cmdlet returns the
volume that it adds. Note, the **Path** parameter must be specified as a drive letter followed by
`WindowsApps` as the directory. Not using this format could lead to inconsistent behavior in the
application model subsystems or the volume itself; for more information see the examples section.
```